ebos/eclcpgridvanguard.hh

```diff
@@ -257,9 +257,13 @@ public:
 protected:
     void createGrids_()
     {
+#ifdef ENABLE_3DPROPS_TESTING
+        const auto& fp = this->eclState().fieldProps();
+        const auto& porv = fp.porv(true);
+#else
         const auto& gridProps = this->eclState().get3DProperties();
         const std::vector<double>& porv = gridProps.getDoubleGridProperty("PORV").getData();
-
+#endif
```

Comment:
Does this need a PR in opm-grid? Shouldn't the size of porv differ and call for a different processing in `grid_->processEclipseFormat`?

Reply:
It is *very* much WIP; right now I am just trying to get it to compile ... ;-(